src/pages/i18n/index.tsx

```diff
@@ -167,15 +188,31 @@ export const I18N = (): JSX.Element => {
             </Toolbar>
 
             <List id={'item-list'}>
-                {Object.keys(fruits).map((fruit, index) => (
+                {fruits.map((fruit, index) => (
                     <InfoListItem
                         key={index}
-                        onClick={(): void => selectFruit(fruit)}
+                        onClick={(): void => selectFruit(fruit.name)}
                         ripple={true}
                         style={{ textAlign: isRTL() ? 'right' : 'left' }}
-                        title={t(`FRUITS.${fruit}`)}
-                        subtitle={t('MORE_INFO')}
-                        icon={<Checkbox checked={selectedItems.has(fruit)} onChange={(): void => selectFruit(fruit)} />}
+                        title={t(`FRUITS.${fruit.name}`)}
+                        subtitle={[
+                            <NumberFormat
```

Comment:
Is there a more global (e.g. theme-level, or t-function-esque) way to do this? Seems like using a component for every number you would need to show in the UI could get overwhelming.

Reply:
Looks like i18next might support something like this (https://www.i18next.com/translation-function/formatting)

Reply:
Ok I'll look into how we can use this formatting and see if it's simpler to use.

Reply:
Just wondering...I know in Material-UIs example they use the numberFormat library, so it's not wrong. Just wondering if it's the simplest way (the PX Blue _recommended_ way)

Reply:
I'm making some progress getting the i18n currency stuff working, but now the hardest part is formatting each currency.   This work was originally handled by the NumberFormat library.  I'll show some snippets of what I've got.

Reply:
Each translation file requires a new entry:
```
// Price is provided through interpolation, currency is a `format` value used in the i18n interpolation fn.
CURRENCY: `{{price, currency}}`,
```


i18n.tsx
This is where the translation/currency conversion occurs.
```
interpolation: {
            format: (value, format, lng) => {
                if (format === 'currency') {
                    if (lng === 'en') {
                        return `$${value}`;
                    }
                } else {
                    return value;
                }
            }
        },
```

index.tsx
Language and price is passed into the interpolation function found above.
`subtitle={t('CURRENCY', { price: fruit.price, lng: lang })}`

Reply:
The work required to get this translation to work smoothly is formatting each locale's currency correctly.